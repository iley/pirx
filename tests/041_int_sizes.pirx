func main(): int {
  var i32: int;
  var i64: int64;

  i32 = 2147483647; // 2^31-1
  i64 = 2147483647l; // 2^31-1

  i32 = i32 + 1;
  i64 = i64 + 1l;

  printf("i32 = %ld i64 = %lld\n", i32, i64);
  return 0;
}
