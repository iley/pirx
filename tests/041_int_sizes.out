i32 = -2147483648 i64 = 2147483648
